app.js

```diff
@@ -1 +1,50 @@
-app.js
\ No newline at end of file
+// task1
+
+let num = [2, 3, 8, 1, 6];
+let c;
+
+for (let i = 0; i < num.length; i++) {
```

Comment:
please create function that getting two argument (array) and return one array
and this is wrong when you will have two the same number you will get wrong array like this
const num1 = [1, 2, 3, 4, 6, 60, 60];
const num2 = [60, 7, 8, 9, 5, 11, 12, 13, 45, 6, 1];

output -> [1, 6, 60, 60]